hydroflow/src/scheduled/mod.rs

```diff
@@ -1,3 +1,5 @@
+use std::fmt;
```

Comment:
Might want to either just import fully `use std::fmt::{Display, Formatter}` or just use `std::` prefix in the impl below

Reply:
Done